### ISSUE TITLE
HTML overlay caching linked files (such as CSS) even if "Do NOT cache this menu..." is selected in action settings

### ISSUE THREAD

**Issue by andyconlin** [folivoraAI/BetterTouchTool#2636]
**Describe the bug**
HTML overlays seem to be caching linked files (CSS, in my case), even if the "Do NOT cache this menu..." is checked in settings.  Changes made to the initial file (the one whose path is inserted in the action settings) will be seen right away, but changes to the linked file are not seen until the menu is manually reloaded.

**Affected input device (e.g. MacBook Trackpad, Magic Mouse/Trackpad, Touch Bar, etc.):**
N/A

**Screenshots**
+ First edit is made to `index.html` (the initial file included in the action settings) and changes are seen the next time the overlay is triggered.
+ Second edit is made to the linked file `style.css` and is not made live on the overlay until it is manually reloaded.
+ Once reloaded, the new styles are seen in future overlay instances.

![htmloverlayreload](https://user-images.githubusercontent.com/24281063/39659555-f42776f6-4fe7-11e8-9c49-3541f13c83fc.gif)

**Device information:**
- Type of Mac: "MacBook Pro (13-inch, 2017, Four Thunderbolt 3 Ports)"
- macOS version: 10.13.4
- BetterTouchTool version: 2.467

**Additional information (e.g. StackTraces, related issues, screenshots, workarounds, etc.):**
_(I also made changes to a linked page (a second html page linked with `<a href...`) and those changes showed up immediately, so it seems to be either CSS specifically, or perhaps any linked file (e.g., javascript, etc.).  I have not tested with other file types.)_

**Comment by fifafu** [comment]
That's true, the caching BTT refers to is about the view itself (which takes a while to prepare, which is why I added this).
The HTML/CSS caching is still handled by the web view.

You could integrate this little Java Script if you want to reload all CSS everytime you open the webview:

```HTML
<html>
<head>
  <script>
    // this function is called everyime you open the menu in BTT
    function BTTInitialize() {
//this will get all the link tags and update their source with a timestamped url
var links = document.getElementsByTagName("link"); for (var i = 0; i < links.length;i++) { var link = links[i]; if (link.rel === "stylesheet") {link.href += "?t="+Date.now(); }}
    }

  </script>
  </head>
```
